lib/datasource/remote/api/rates_repository.dart

```diff
@@ -16,15 +16,34 @@ class RatesRepository extends NetworkRepository {
         .catchError((error) => mapHttpError(error));
   }
 
-  Future<Result<RateModel>> getUSDRate() async {
+  Future<Result<RateModel>> getSeedsRate() async {
     print('[http] get seeds rate USD');
 
     final request = '{"json":true,"code":"tlosto.seeds","scope":"tlosto.seeds","table":"price"}';
 
     return http
         .post(Uri.parse('$baseURL/v1/chain/get_table_rows'), headers: headers, body: request)
         .then((http.Response response) => mapHttpResponse<RateModel>(response, (dynamic body) {
-              return RateModel.fromJson(body);
+              return RateModel.fromSeedsJson(body);
+            }))
+        .catchError((error) => mapHttpError(error));
+  }
+
+  Future<Result<RateModel>> getTelosRate() async {
+    print('[http] get telos rate USD');
+
+    final params = createRequest(
+      code: "delphioracle",
```

Comment:
what happens when we need to make another call that uses 'delphioracle' ? 

Do we duplicate the string there? What if we make 10 more calls that use delphioracle? 

I am not going to block this PR for this. 

But ill make a follow up PR to ensure this is not possible anymore. "ill make createRequest function take a typed and not any string. "

Reply:
I think this is specific to this call and should not be abstracted out or shared with other code.

But you could see it either way. 

I guess it is kind of like a URL, so maybe should be in constant strings somewhere.